src/blockchain_db/blockchain_db.h

```diff
@@ -1083,6 +1083,22 @@ class BlockchainDB
    */
   virtual uint64_t get_tx_unlock_time(const crypto::hash& h) const = 0;
 
+  // return unlock time of output with the given amount and output amount index
+  /**
+   * @brief fetch an output's unlock time/height
+   *
+   * The subclass should return the stored unlock time for the output
+   * with the given amount and output amount index.
+   *
+   * If no such output exists, the subclass should throw OUTPUT_DNE.
+   *
+   * @param amount the amount of the requested output
+   * @param amount_index the amount index of the requested output
```

Comment:
Nit: the comment doesn't match the parameter name (`amount_index` vs `global_index`)